docpages/example_programs/the_basics/using_callback_functions.md

```diff
@@ -79,11 +75,18 @@ int main() {
 
     bot.on_ready([&bot](const dpp::ready_t& event) {
         if (dpp::run_once <struct register_global_commands>()) {
-            dpp::slashcommand msg_get("msg-get", "Get the message", bot.me.id);
+            dpp::slashcommand msgs_get("msgs-get", "Get messages", bot.me.id);
+
+            msgs_get.add_option(
+                dpp::command_option(dpp::co_integer, "quantity", "Quantity of messages to get. Max - 100.")
+                .set_max_value(100)
+                .set_min_value(0)
+            );
+
             dpp::slashcommand channel_create("channel-create", "Create a channel", bot.me.id);
             dpp::slashcommand msg_error("msg-error", "Get an error instead of message :)", bot.me.id);
 
-            bot.global_bulk_command_create({msg_get, channel_create, msg_error});
+            bot.global_bulk_command_create( {msgs_get, channel_create, msg_error} );
```

Comment:
`bot.global_bulk_command_create({ msgs_get, channel_create, msg_error });` as per the style guidelines

Reply:
I already did it but maybe need to update pr idk

Reply:
You corrected it to `( {msgs_get...` not `({ msgs_get...`, [here's the link again so you can read what I asked for originally](https://dpp.dev/coding-standards.html#:~:text=braces%20with%20statements!-,Lists,-Lists%20should%20have) :)

Reply:
RAHHHH i forgor :(

Reply:
Will b don when i m hom

Reply:
Perfect now (works on my machine only)